migtests/tests/pg/assessment-report-test/pg_assessment_report.sql

```diff
@@ -346,4 +348,16 @@ BEGIN
     RAISE NOTICE 'Updated record with ID: %, CIDR: %, BIT: %, Date range: %', 
         p_id, p_c, p_bitt, p_d;
 END;
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER t_raster BEFORE UPDATE OR DELETE ON public.combined_tbl
+    FOR EACH ROW EXECUTE FUNCTION lo_manage(raster);
```

Comment:
Q: how is this case being detected rightnow, which codepath...

Reply:
In the TriggerIssuesDetector in `detectors_ddl.go`